scripts/install-binaries

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+REV=a6a3290833f6da4a7785a426bed342df069ce47f
+
+BINARIES=(rad radicle-node radicle-httpd git-remote-rad)
+REPO_ROOT=$(git rev-parse --show-toplevel)
+BINARY_PATH=$REPO_ROOT/tests/tmp/bin/${REV:0:7}
+OS=$(uname)
+
+mkdir -p "$BINARY_PATH"
+
+for BINARY_NAME in "${BINARIES[@]}"
+do
+  # Handle different naming of binaries than required locally
```

Comment:
I think we can improve this comment by saying here why we need `rad-debug` for tests rather than the production `rad`. Nit: I try to end any comment with a full stop everywhere.

Reply:
Ok changed it to
```
# To provide deterministic Node and Repo IDs, we need a rad CLI compiled with the --debug flag.
```